### PR TITLE
fix: missing options property in schema and ID's should be strings

### PR DESCRIPTION
**Pull request by Photonios**
An explanation for this in detail: https://github.com/webpack-contrib/multi-loader/issues/10#issuecomment-379463986

### CODE REVIEW

**Comment by bjankord** [comment]
This looks like a decent PR to get into the project @evilebottnawi @michael-ciniawsky 
It's small and [resolves some issues using extract-text-plugin to generate multiple theme files with combine-loader and multi-loader](https://github.com/webpack-contrib/multi-loader/issues/10#issuecomment-379463986)

**Comment by gokulkrishh** [comment]
@Photonios @sokra Is there any plan to merge this PR ??

**Comment by michael-ciniawsky** [comment]
The commits need either be reworded or this PR needs to be splitted into 2 separate PR's please, since we generate the `CHANGELOG` based on the commit history

```
fix(schema/plugin): add missing `id` option
fix(index): id's should be strings (`options.id`)
```

File: src/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ class ExtractTextPlugin {
       validateOptions(path.resolve(__dirname, '../schema/plugin.json'), options, 'Extract Text Plugin');
     }
     this.filename = options.filename;
-    this.id = options.id != null ? options.id : ++nextId;
+    this.id = (options.id != null ? options.id : ++nextId).toString();
```
Comment (michael-ciniawsky) [inline src/index.js]:
```suggestion
    this.id = String(options.id != null ? options.id : ++nextId);
```

**Comment by gokulkrishh** [comment]
@michael-ciniawsky Thanks!!

**Comment by gokulkrishh** [comment]
@Photonios Hey are planning to do the changes or shall I take it up ?? Let me know thanks!!

**Comment by liamsteele** [comment]
@gokulkrishh Would you be able to progress this issue if @Photonios doesn't? This would be a great fix to have available for me.

**Comment by Photonios** [comment]
Hello everyone,

Sorry for the long wait. Other things got in the way.

I've reworded the commits as @michael-ciniawsky suggested and applied his suggestion for the string conversion.

Cheers

**Comment by alexander-akait** [comment]
DEPRECATED
Please use: https://github.com/webpack-contrib/mini-css-extract-plugin

**Pull request closed without merging**